bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/utils/SvnUtil.java

```diff
@@ -15,527 +15,545 @@
  */
 package org.aposin.mergeprocessor.utils;
 
+import com.google.common.base.Joiner;
+
+import org.apache.commons.io.FileUtils;
+import org.aposin.mergeprocessor.application.ApplicationUtil;
+import org.aposin.mergeprocessor.configuration.Configuration;
+import org.aposin.mergeprocessor.exception.SvnUtilException;
+import org.aposin.mergeprocessor.model.svn.ISvnClient;
+import org.aposin.mergeprocessor.model.svn.ISvnClient.SvnClientException;
+import org.aposin.mergeprocessor.model.svn.SVNMergeUnit;
+import org.aposin.mergeprocessor.view.MessageDialogScrollable;
+import org.eclipse.e4.ui.di.UISynchronize;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.osgi.util.NLS;
+
 import java.io.File;
 import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
-
-import org.apache.commons.io.FileUtils;
-import org.aposin.mergeprocessor.application.ApplicationUtil;
-import org.aposin.mergeprocessor.configuration.Configuration;
-import org.aposin.mergeprocessor.model.svn.ISvnClient;
-import org.aposin.mergeprocessor.model.svn.ISvnClient.SvnClientException;
-import org.aposin.mergeprocessor.model.svn.SVNMergeUnit;
-import org.aposin.mergeprocessor.view.MessageDialogScrollable;
-import org.eclipse.e4.ui.di.UISynchronize;
-import org.eclipse.jface.dialogs.MessageDialog;
-import org.eclipse.osgi.util.NLS;
-
-import com.google.common.base.Joiner;
-
 /**
  * 
  */
 public class SvnUtil {
 
-	private static final Logger LOGGER = Logger.getLogger(SvnUtil.class.getName());
```

Comment:
You exchanged spaces for tabs, so the whole file has changed.
Please have a look to your Java Formatter, as we only want to use tabs only (see screenshot):
![grafik](https://user-images.githubusercontent.com/6864222/67655429-de130880-f950-11e9-8e7e-8a8feaffb36e.png)